examples/socketio-webcam-videostream/pycli/cli.py

```diff
@@ -0,0 +1,108 @@
+import asyncio
+import platform
+import socketio
+
+import cv2
+
+from aiortc import (
+    RTCIceCandidate,
+    RTCPeerConnection,
+    RTCSessionDescription,
+)
+from aiortc.contrib.media import MediaPlayer
+from aiortc.contrib.signaling import object_from_string, object_to_string
+
+# Globals
+sio = socketio.AsyncClient()
+pc = None
+is_initiator = False
+
+def add_tracks():
+    # Webcam to record from
+    options = {"framerate": "30", "video_size": "640x360"}
+    if platform.system() == "Darwin":
+        webcam = MediaPlayer(
+            "default:none", format="avfoundation", options=options
+        )
+    elif platform.system() == "Windows":
+        webcam = MediaPlayer(
+            "video=Integrated Camera", format="dshow", options=options
+        )
+    else:
+        webcam = MediaPlayer("/dev/video0", format="v4l2", options=options)
+    if webcam and webcam.audio:
+        pc.addTrack(webcam.audio)
+
+    if webcam and webcam.video:
+        pc.addTrack(webcam.video)
+
+async def createOffer():
+    print("Creating an offer. Waiting for an answer")
```

Comment:
Use `logging` class to print message and add `--verbose` `arg` option in your example.